CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,70 @@
+# Tinted Theming Code of Conduct
+
+## Our Commitment
+
+Tinted Theming is committed to providing a welcoming and inclusive community where everyone feels respected and valued. We expect all community members, including contributors, maintainers, and users, to adhere to this Code of Conduct.
+
+## Inclusivity and Respect
+
+We value diversity and believe that a wide range of perspectives and experiences enriches our community. We are committed to creating an environment free from discrimination, harassment, and disrespectful behavior.
+
+## Language and Communication
+
+We strive for a community where everyone communicates with respect and empathy. Language has power, and we must use it responsibly.
+
+### Understanding Slurs and Offensive Language
+
+It's important to be aware that certain words, even those that might seem commonplace, can be harmful. This includes slurs or derogatory language directed at any disadvanged and underrepresented group, whether based on race, ethnicity, gender, disability, sexual orientation, or other characteristics. Such language:
+
+- Contributes to a culture of exclusion.
+- Creates an unwelcoming environment for affected individuals and groups.
+- Contradicts our values of respect and inclusivity.
+
+## Expected Behavior
+
+We expect all members of the Tinted Theming community to:
+
+1. Be respectful and considerate of others, regardless of their background, identity, or beliefs.
+2. Use inclusive language and avoid derogatory or offensive comments. This includes refraining from any slurs or derogatory language. If you're unsure whether a word might be offensive, we encourage erring on the side of caution and choosing more neutral terms.
+3. Listen actively and be open to different viewpoints and ideas.
+4. Provide constructive feedback and criticism in a respectful manner.
+5. Be patient and supportive, especially with newcomers.
+6. Respect and protect the privacy of others.
+7. Respect others' pronouns and gender identities. Avoid making assumptions about anyone's gender identity. Use gender-neutral language or the pronouns that individuals have specified for themselves.
+8. Educate themselves on why certain words are harmful and strive to use language that respects all community members. Be open to learning and growing in their understanding of respectful communication. We also encourage open and respectful discussion about language and its impact, as we all play a role in creating a supportive community environment.
```

Comment:
I like the intent here, but this seems relatively heavy handed when we've also got bullet points for slurs and preferred pronouns. Was there anything in particular this bullet point is aiming to address? Could we rework it to be more focused?

Reply:
The goal with this was to encourage people to learn what slurs are so they don't use them unintentionally.  I hear/see people use a slur for mentally disabled people as an insult relatively commonly. I can see how this point comes across as verbose or redundant though. I'll remove it since I feel like it's covered by:

```markdown
## Language and Communication

We strive for a community where everyone communicates with respect and empathy. Language has power, and we must use it responsibly.
```